packages/docs/docs/guides/3-API/0-portal-intro.md

```diff
@@ -16,6 +16,23 @@ Our API accepts JSON-encoded request bodies, returns JSON-encoded responses, and
 
 Aside the guide, you can also consult [the API documentation](https://api-docs.request.network/).
 
+
+# Authentication
+
+All API endpoints are authenticated.
+Two mechanisms are currently allowed:
+
+- API Key, explained below, to be used for scripting and test purposes
+- OAuth, explain in the [Apps](3-api-apps) section
+
+:::important
+API Keys don't enable the best security and UX for your users, we might drop their support in the future.
```

Comment:
I agree we should have a disclaimer here but maybe it should be reworded? I feel it could potentially scare people.

Reply:
indeed, any idea?

Reply:
Maybe we should say

At Request we are always reviewing our security policy and following the best security practises. Because of this, there is a possibility we may deprecate usage of API keys in the future. 

In general though, I don't think this warning is really necessary.